app/assets/v2/js/pages/bounty_details.js

```diff
@@ -174,9 +174,13 @@ var callbacks = {
         }
         return [ label , response];
     },
-    
+
 }
 
+var isBountyOwner = function(result) {
+    var bountyAddress = result['bounty_owner_address']
+    return (web3 && (web3.eth.coinbase == bountyAddress))
```

Comment:
make sure you make this comparison case insensitive!

Reply:
There are other places in the codebase where we aren't doing case-insensitive comparison checks (for example, [here](https://github.com/gitcoinco/web/blob/32d8f4519e7b5b9d4133d6f1766794ba41329c4b/app/assets/v2/js/pages/dashboard.js#L175)). If you want, I can probably find all these instances and replace them with case-insensitive comparison cases...I just want to confirm first to make sure that case-insensitive comparison cases is indeed what we want.

Reply:
ah shoot.. i didnt realize this. 

i dont think making these updates as part of this PR makes sense. thanks for the offer tho.